src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println(FigureSupplier.getDefaultFigure());
+        for (int i = 0; i < 10; i++) {
+            System.out.println(FigureSupplier.getRandomFigure());
+        }
```

Comment:
your task is:
The first half of figures in this array should be generated with random parameters.
The other half of the figures should have the same, default parameters.